README.md

```diff
@@ -51,14 +51,14 @@ Internally I just had to combine EventEmitter and use the first argument to pass
 ### Master
 
 ```javascript
-  var server = fork('some/node/app.js')
-  var client = IPCEE(server)
+  const server = fork('some/node/app.js')
+  const client = IPCEE(server)
 
-  client.once('started', function() {
```

Comment:
It's not a good idea to change the function context on events. I don't remember the exact cause but I know it's a bug source.